app/models/api_gateway_integration.rb

```diff
@@ -0,0 +1,7 @@
+class ApiGatewayIntegration < ApplicationRecord
+  belongs_to :project
+
+  validates :title, presence: false
+  validates :uri_prefix, presence: false
+  validates :timeout_in_millis, presence: false
```

Comment:
If you wanted to say you don't want to check the presence (so allow nil), you simply can remove those 3 lines.